torch/fx/experimental/symbolic_shapes.py

```diff
@@ -6082,6 +6087,12 @@ def _evaluate_expr(
 
         # TODO: split conjunctions and evaluate them separately
 
+        if isinstance(
+            orig_expr,
+            (sympy.logic.boolalg.BooleanTrue, sympy.logic.boolalg.BooleanFalse),
+        ):
+            return orig_expr
```

Comment:
This is fine but it's weird that we never ran into this before. In particular, I would have expected there to be some preexisting logic to circuit out when orig_expr is just a constant.